unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/PlayerName/PlayerName.cs

```diff
@@ -186,7 +188,7 @@ private async UniTaskVoid AsyncSetName(string name, bool isClaimed, bool isGuest
 
     private async UniTask<string> FilterName(string name) =>
         IsProfanityFilteringEnabled()
-            ? await ProfanityFilterSharedInstances.regexFilter.Filter(name)
+            ? await Environment.i.serviceLocator.Get<IProfanityFilter>().Filter(name)
```

Comment:
can this one be cached? (or in ideal case `PlayerName` get's reference to `IProfanityFilter` via `Inizialize` method, but I didn't found any client of this code, who can do it. Maybe you know one)

Reply:
I've cached it. I find a better approach to refactor into MVC instead of creating the initialize method. Its actually being discussed how is the best way to do it